src/Microsoft.OData.Core/ODataServiceColectionExtensions.cs

```diff
@@ -0,0 +1,58 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ODataServiceColectionExtensions.cs" company="Microsoft">
+// Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System;

```

Comment:
The style in this project appears to be to nest usings inside the namespace block

Reply:
Please do not put usings inside the namespace block. It's an unnecessary readability hit.

Reply:
@gathogojr